lib/topological_inventory/ansible_tower/collector.rb

```diff
@@ -93,20 +93,24 @@ def collector_thread(connection, entity_type)
         sweep_scope.merge(parser.collections.values.map(&:name))
       end
 
-      logger.info("[END] Collecting #{entity_type} with :refresh_state_uuid => '#{refresh_state_uuid}' - Parts [#{total_parts}]")
+      logger.info("[END] Collecting #{entity_type}, :source_id => #{source_id}, :refresh_state_uuid => '#{refresh_state_uuid}' - Parts [#{total_parts}]")
 
       # Sweeping inactive records
       sweep_scope = sweep_scope.to_a
-      logger.info("[START] Sweeping inactive records for #{sweep_scope} with :refresh_state_uuid => '#{refresh_state_uuid}'...")
+      logger.info("[START] Sweeping inactive records for #{sweep_scope}, :source_id => #{source_id}, :refresh_state_uuid => '#{refresh_state_uuid}'...")
       sweep_inventory(inventory_name, schema_name, refresh_state_uuid, total_parts, sweep_scope, refresh_state_started_at)
-      logger.info("[END] Sweeping inactive records for #{sweep_scope} with :refresh_state_uuid => '#{refresh_state_uuid}'")
+      logger.info("[END] Sweeping inactive records for #{sweep_scope}, :source_id => #{source_id}, :refresh_state_uuid => '#{refresh_state_uuid}'")
     rescue => e
       metrics.record_error
-      logger.error("Error collecting: #{entity_type} :refresh_state_uuid => #{refresh_state_uuid}, message => #{e.message} #{e.backtrace.join("\n")}")
+      logger.error("Error collecting: #{entity_type}, :source_id => #{source_id}, :refresh_state_uuid => #{refresh_state_uuid}, message => #{e.message} #{e.backtrace.join("\n")}")
     end
 
     def inventory_name
       "AnsibleTower"
     end
+
+    def source_id
+      @source_id ||= Settings.sources.select { |s| s.source == source }&.first&.source_id
```

Comment:
@lindgrenj6 Collector doesn't initialize Config thus it doesn't know Settings so it can't work in standalone mode.

Reply:
Thanks for catching that, I updated it to log the Source UID since we'll always have that field.